.travis/run.sh

```diff
@@ -24,5 +24,12 @@ else
         pyenv global pypy-2.6.0
     fi
 fi
+
+# If we're building on Python 3.5, then we need to use plain asserts until
+# pytest is updated.
```

Comment:
https://bitbucket.org/pytest-dev/pytest/issue/744/pytest-assert-rewriting-broken-with-python is the issue